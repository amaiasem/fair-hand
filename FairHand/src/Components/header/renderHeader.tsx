import React from 'react'
import { StyleSheet, View, Image } from 'react-native'
import { COLOR, SIZES, images, SHADOW } from '../../../constants'

const styles = StyleSheet.create({
  headerBox: {
    flexDirection: 'row',
    height: 50,
    width: SIZES.width,
    backgroundColor: COLOR.white,
    ...SHADOW

  },
  logo: {
    marginTop: 3,
    width: 150,
    height: 40
  }
})

function renderHeader () {
  return (
      <View style={styles.headerBox}>
        <Image style={styles.logo} source={images.blackLogo}></Image>
      </View>
  )
}

export default renderHeader
