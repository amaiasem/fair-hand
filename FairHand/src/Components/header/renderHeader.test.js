import React from 'react'
import renderHeader from './renderHeader'
import { render } from '@testing-library/react-native'

describe('Given a component renderHeader', () => {
  const component = (<renderHeader/>)
  test('Then should have onechild', () => {
    render(component)
    expect(component).toMatchSnapshot()
  })
})
