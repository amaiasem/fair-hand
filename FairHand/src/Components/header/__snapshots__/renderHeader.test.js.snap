// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a component renderHeader Then should have onechild 1`] = `<renderHeader />`;
